dev/tools/update_icons.dart

```diff
@@ -513,12 +513,14 @@ class Icon {
 
   String get usage => 'Icon($className.$flutterId),';
 
-  String get mirroredInRTL => _iconsMirroredWhenRTL.contains(shortId)
-      ? ', matchTextDirection: true'
-      : '';
+  bool get isMirroredInRTL {
+    // Remove common suffixes (e.g. "_new" or "_alt") from the shortId.
+    final String normalizedShortId = shortId.replaceAll(RegExp(r'_(new|alt|off|on)$'), '');
```

Comment:
Should we also add test cases for "_off" and "_on" as they are mentioned here:)

Reply:
sure! thanks